diff --git a/contrib/libcss/src/parse/properties/parser_wrapper.sh b/contrib/libcss/src/parse/properties/parser_wrapper.sh
index 2ae29e8..32f86b1 100755
--- a/contrib/libcss/src/parse/properties/parser_wrapper.sh
+++ b/contrib/libcss/src/parse/properties/parser_wrapper.sh
@@ -1,4 +1,4 @@
-#!/usr/bin/bash
+#!/usr/bin/env bash
 
 if [[ $# -lt 3 ]]; then
        echo "Requires at least three arguments (output directory, gen_parser binary location, properties.gen location)"
